src/adapters/mc/OsConfigPolicy.mof

```diff
@@ -1,10 +1,11 @@
 instance of OsConfigResource as $OsConfigResource0ref
 {
-    ResourceID = "Ensure nodev option set on /home partition";
-    PayloadKey = "EnsureNodevOptionOnHomePartition";
+    ResourceID = "Ensure permissions on /etc/ssh/sshd_config are configured";
+    PayloadKey = "EnsurePermissionsOnEtcSshSshdConfig";
     ComponentName = "SecurityBaseline";
-    ReportedObjectName = "auditEnsureNodevOptionOnHomePartition";
-    DesiredObjectName = "remediateEnsureNodevOptionOnHomePartition";
+    ReportedObjectName = "auditEnsurePermissionsOnEtcSshSshdConfig";
+    ExpectedObjectValue = "PASS";

```

Comment:
That's described in the PR title and in the design spec. Expected* frees Desired* so that later can be used for runtime parameter values. In general, recommend to look at the design spec (only 5 pages or so) rather than ask questions which are already answered there.

Reply:
I do not see a description for this PR other than the checklist. It may be good to add a summary to the PR description since not everyone on GitHub has access to the internal design document.

Reply:
It's all in the title.